omod/src/main/web/dashboardwidgets/relationships/relationships.html

```diff
@@ -39,7 +39,7 @@
 
     <li ng-show="$ctrl.edit">
         <p class="left">
-        <select class="relationship-input" ng-options="type as type.name for type in $ctrl.types"
+        <select class="relationship-input" ng-options="type.name | translate for type in $ctrl.types"
```

Comment:
Thanks for the review @mogoodrich.  Since we are not referring to `type` within the `<select/>` element, it is fine as well to directly refer to `type.name` the label. `... type in $ctrl.types` addresses the value assigned to the individual `option`s which is actually the `type` object as you've highlighted. This works fine already as I have tested.

Reply:
Cool, thanks for the explanation, just wanted to double-check!